src/constants/memory/hram.asm

```diff
@@ -499,11 +499,11 @@ hMapId::
   ds 1 ; FFF7
 
 ; Status flags of the current room
-; 0x00 = not visited,
-; 0x04 = bos door is open,
-; 0x10 = special event occured,
-; 0x30 = don't load room entities,
-; 0x80 = visited,
+; 0x00 = not visited
+; 0x04 = boss door is open
+; 0x10 = special event occured
+; 0x30 = boss defeated
```

Comment:
These are slightly "wrong". 0x10, 0x20, 0x40 are just room flags, depending on the entities/events in the room how they are used.

0x80 is used to show the room as visited on the map (both the dungeon map and the overworld map)

0x01, 0x02, 0x04, 0x08 are the 4 door direction doors are open if I remember right, independend of the type of door. Not sure which bit is which direction.

Reply:
Thanks! I corrected the values, and added them to this PR.